region/norfair/east/Norfair Reserve Tank Room.json

```
@@ -407,6 +407,25 @@
         "Place a second Power Bomb while jumping up from the lower item block to clear the remaining enemies for the return trip."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "G-Mode",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        {"or": [
+          "canPauseAbuse",
+          {"heatFrames": 19}
+        ]},
+        {"heatFrames": 0}
```

```suggestion
        {"heatFrames": 19}
```
You still take the heat damage